inventories/README.md

```diff
@@ -7,82 +7,43 @@
-* the physical machines
-* the cluster internal network (Open vSwitch topology)
-* the virtual machines
-## SEAPATH Ansible inventories examples description
+In the `examples` directory, you can find some commented and minimal examples of inventories for a standalone and cluster configuration.
 
-All inventories examples are in YAML format. The example inventories are:
-* `seapath_demo_cluster_definition_example.yml`: SEAPATH cluster demo
-* `seapath_standalone_definition_example.yml`: SEAPATH standalone hypervisor setup
-* The complex SEAPATH cluster inventories are:
-	* `seapath_cluster_definition_example.yml`: physical machines
-	* `seapath_ovstopology_definition_example.yml`: cluster internal network (Open vSwitch topology)
-	* `seapath_vm_definition_example.yml`: virtual machines
+It is possible with Ansible to provides [multiple inventories files](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#passing-multiple-inventory-sources), and we recommend using this feature for complex setup.
+In this directory, we have separated the SEAPATH inventories into :
+* seapath-cluster.yaml : Describe the physical machines in a cluster
+* seapath-standalone.yaml : Describe one standalone physical machine
+* seapath_ovs.yml : Describe an extended network over the cluster (use only in cluster mode)
+* seapath-vm-deployement.yaml : Describe the variable to deploy a VM, either in standalone or cluster mode
 
-The directory also contains the firewall rules `iptables_rules_example.txt` which are referring in `seapath_cluster_definition_example.yml`.
+All these inventories contain only the minimal variables required to configure and run SEAPATH. For advanced configuration, please refer to the wiki page : TODO put link
 
-The simple SEAPATH cluster inventory require only minor tweak to be adapted. In all other files described, all the options which do not match a realistic case and need to be modified are indicated.
+### Cluster inventory
 
-### Optional variables
+This inventory describes a cluster using two hypervisors and one observer. It is possible to use a three-hypervisor setup by adding a machine into the `hypervisors` section and removing the `observers` part.
+TODO : Put link to cluster architecture page on wiki when written
 
-You can find concrete examples of the variables in the inventories of this directory. Below are optional advanced variables that are not described in the examples.
+### OVS inventory
 
-#### Network implementation
+This inventory describes a network Open vSwitch bridge over the cluster. It allows VM to communicate even after a migration toward another hypervisor.
+This work is done through the variable `ovs_bridges`, that is passed to the `seapath-config_ovs` systemd service. This service then calls [python3-setup-ovs](https://github.com/seapath/python3-setup-ovs)
```

Comment:
I am not sure mentioning seapath-config_ovs is a good idea. It's adds confusion.